pkg/build/builder/docker.go

```diff
@@ -69,6 +72,20 @@ func (d *DockerBuilder) Build() error {
 	tag := d.build.Parameters.Output.DockerImageReference
 	defer removeImage(d.dockerClient, tag)
 	if len(d.build.Parameters.Output.DockerImageReference) != 0 {
+		// TODO: This should use ImageRepository ObjectReference in future
```

Comment:
@bparees will remove that TODO, my understanding was that we will refactor this to support referecences, but agree I don't need this :-)